README.md

```diff
@@ -86,155 +87,144 @@ _choose one_ of the following submodule-cloning techniques:
+## Validate your changes
 
-## Pre-push site checks
+### Check documentation and example code
 
-### Checking documentation and example code
-
-If you've made changes to this site's documentation and/or example code,
-and committed locally, then run the following commands before pushing your work:
+If you've made changes to the code in the `/examples` or `/tool` directories,
+commit your work, then run the following command to
+verify it is up to date and matches the site standards.
 
 ```terminal
-# Enter a running Docker container shell
-$ make run
-
-# Check/validate example code
-$ dart run dart_site check-all
+dart run dart_site check-all
 ```
 
-If these scripts report errors or warnings,
-then address those issues and rerun the above commands.
-Otherwise, you can push your changes.
+If this script reports any errors or warnings,
+then address those issues and rerun the command.
+If you have any issues, reach out for help.
 
-## Deploying to a staging site
+## Deploy to a staging site
 
-You can deploy your local edits to a
-personal Firebase hosting staging site as follows:
+Submitted pull requests can be automatically staged
+by a site maintainer.
+If you'd like to stage the site yourself though,
+you can build a full version and upload it to Firebase.
 
 1. If you don't already have a Firebase project,
 
    - Navigate to the [Firebase Console](https://console.firebase.google.com)
      and create your own Firebase project (for example, `dart-dev-staging`).
 
-   - Head back to your local repo shell and verify that you are logged in.
+   - Head back to your local terminal and verify that you are logged in.
 
      ```terminal
-     $ firebase login
+     firebase login
      ```
 
    - Ensure that your project exists and activate that project:
 
      ```terminal
-     $ firebase projects:list
-     $ firebase use <your-project>
+     firebase projects:list
+     firebase use <your-project>
      ```
 
-1. Build the site via Docker:
+2. From the root directory of the repository, build the site:
 
    ```terminal
-   $ make build
+   dart run dart_site build
    ```
 
    This will build the site and copy it to your local `_site` directory.
    If that directory previously existed, it will be replaced.
 
-1. Deploy to your activated Firebase project's default hosting site:
+3. Deploy to your activated Firebase project's default hosting site:
 
    ```terminal
-   $ FIREBASE_PROJECT=<your-project> make deploy
+   firebase deploy --only hosting
    ```
 
-   > **TIP:** Add your `FIREBASE_PROJECT` env var to your `.env` file
-   > and it will overwrite the default every time you deploy without specifying.
-
-1. Navigate to your PR on GitHub and update it with the location of
-  the staged version, the names of your reviewers, and so on.
+4. Navigate to your PR on GitHub and include the link of the staged version.
+   Do consider adding a reference to the commit you staged,
+   so that reviewers know if any further changes have been made.
```

Comment:
doesn't our staging GitHub build handle this?

Reply:
Yes it does. This entire section is for if you'd like to stage on your own or you need to for some reason.

Reply:
Can we perhaps add an [Optional] to the title intro section at the top title? Want to flag this is not required and folks should only do it if they want to for their own purposes.